packages/@aws-cdk/aws-apigateway/lib/stage.ts

```diff
@@ -262,8 +297,447 @@ export class Stage extends Resource {
         loggingLevel: options.loggingLevel,
         metricsEnabled: options.metricsEnabled,
         throttlingBurstLimit: options.throttlingBurstLimit,
-        throttlingRateLimit: options.throttlingRateLimit,
+        throttlingRateLimit: options.throttlingRateLimit
       };
     }
   }
 }
+
+/**
+ * A API Gateway custom access log destination.
+ */
+export interface IAccessLogDestination {
```

Comment:
Let's move this and all new classes and interfaces around access log into a separate file - `access-log.ts`